src/main/java/com/notably/logic/commands/OpenCommand.java

```diff
@@ -0,0 +1,21 @@
+package com.notably.logic.commands;
+
+import com.notably.commons.core.path.Path;
+import com.notably.logic.commands.exceptions.CommandException;
+import com.notably.model.BlockManager;
+
+/**
+ * Represents a command that opens a path.
+ */
+public class OpenCommand extends Command {
+    public static final String COMMAND_WORD = "open";
+
+    private final Path path;
+    public OpenCommand(Path path) {
+        this.path = path;
+    }
+    public void execute(BlockManager notablyModel) throws CommandException {
```

Comment:
You don't really need to make `execute` `throws CommandException` here!
Also, let's add the `@Override` annotation above `execute`

Reply:
Will need to throw an exception when i change from `RelativePath` to a  `AbsolutePath`. Will add a TODO comment instead.